app/views/spree/shared/_review.html.erb

```diff
@@ -1,29 +1,26 @@
-<div class="review" itemprop="reviews" itemscope itemtype="http://schema.org/Review">
```

Comment:
Please put back these on main div.

As its related to google seos.

itemprop="reviews" itemscope itemtype="http://schema.org/Review"